backend/commons/src/main/java/org/sonarsource/sonarlint/core/commons/util/gitblame/GitBlameUtils.java

```diff
@@ -79,7 +79,7 @@ private static UnaryOperator<String> adaptToPlatformBasedPath(UnaryOperator<Stri
     };
   }
 
-  private static Repository buildGitRepository(Path basedir) {
+  public static Repository buildGitRepository(Path basedir) {
```

Comment:
Since GitUtils is more general, I think this method better fits GitUtils. Do you agree to move it there?

Reply:
Fixed in the last commit (34ae30b02d40dedea6c77724771076313aa9ef49)